docs/governance.rst

```diff
@@ -0,0 +1,123 @@
+Binder Project Governance
+=========================
+
+The following sections describe governance and high-level principles
+surrounding the Binder Project.
+
+Project mission
+~~~~~~~~~~~~~~~
+
+The mission of the Binder Project is to create, advance, and promote
+open technology that makes it easy for people to connect their data
+science communication, education, and scientific reproducibility work
+with computational environments which can run their work.
+
+The Binder Project pursues this mission by advancing the tools
+surrounding BinderHub, an open-source tool for hosting ad-hoc
+environments in the cloud. The project also serves as the primary
+maintainer of ``mybinder.org``, a public service and demonstration of
+the BinderHub technology.
+
+Membership
+~~~~~~~~~~
+
+Membership in the Binder Project is open to anyone who is nominated by
+an existing member. Members form the trusted group of individuals who
+manage the Binder Project.
+
+Membership comes with `privileges`_ and `responsibilities`_.
+
+Currently, the Binder Project team overlaps 100% with `the BinderHub
+team`_. However, these two groups may diverge in their membership over
+time.
+
+The authoritative list of members and their roles (alphabetical sorting):
+* Tim Head (red)
+* Min Ragan-Kelley (team leader)
```

Comment:
```suggestion
* Min Ragan-Kelley (team leader)
* Erik Sundell (blue)
```

Reply:
I don't understand how to avoid cluttering a line above or below though using this "review comment suggestions" feature.

Reply:
I think this worked?! I clicked commit and it seems to have done what I was expecting it to do.